firmware/controllers/algo/fuel/dfco.cpp

```diff
@@ -24,13 +26,17 @@ bool DfcoController::getState() const {
 	bool rpmActivate = (rpm > engineConfiguration->coastingFuelCutRpmHigh);
 	bool rpmDeactivate = (rpm < engineConfiguration->coastingFuelCutRpmLow);
 
-	// RPM is high enough, and DFCO allowed
-	if (dfcoAllowed && rpmActivate) {
+	// greater than or equal so that it works if both config params are set to 0
```

Comment:
"Helps with low-speed jerkiness from decel fuel cut in low gears." can we please make this code comment not PR comment?